### ISSUE TITLE
Can you please share the real videos in the test set?

### ISSUE THREAD

**Issue by maudzung** [phlong3105/aic22_track4#1]
Hi @phlong3105 
Thanks for your work. Can you please share the real videos in the test set? I didn't join the competition, so I can't access to download the test videos at the moment.
Thank you once again!

**Comment by longph3105** [comment]
Training data and test videos are included inside the docker image. You can easily obtain them there.

**Comment by maudzung** [comment]
Thank you so much for your information.
